### ISSUE TITLE
hooksecurefunc(): SetTemplate is not a function

### ISSUE THREAD

**Issue by Florian-Wegner** [dlecina/ElvUI_NoBorders#4]
WoW TWW: 11.0.2
Addon: 1.2.2

```
167x ElvUI_NoBorders/categories/misc/NoBorders.lua:56: hooksecurefunc(): SetTemplate is not a function
[string "=[C]"]: ?
[string "@ElvUI_NoBorders/categories/misc/NoBorders.lua"]:56: in function `addapi'
[string "@ElvUI_NoBorders/categories/misc/NoBorders.lua"]:68: in main chunk

Locals:
```

**Comment by dlecina** [comment]
Fixed in 1.2.3

**Comment by chrismaille** [comment]
Can confirm this issue on v1.2.3. Made a PR: #6
